model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -2236,13 +2236,15 @@ def _infer_hardware(
     llm_artifact_gateway: LLMArtifactGateway,
     model_name: str,
     checkpoint_path: str,
+    is_batch_job: bool = False,
 ) -> CreateDockerImageBatchJobResourceRequests:
     config = llm_artifact_gateway.get_model_config(checkpoint_path)
 
     dtype_size = 2
+    kv_multiplier = 5 if is_batch_job else 2
```

Comment:
can you add tests to ensure llama 3 70b use 4 GPUs?

Reply:
done